pipeline/transform.py

```diff
@@ -0,0 +1,145 @@
+"""Script for transforming data from the Bandcamp API"""
+import logging
+import re
+from datetime import datetime
+import requests
+import pandas as pd
+from bs4 import BeautifulSoup
+from extract import main_extract
+
+
+def config_log() -> None:
+    """Terminal logs configuration"""
+    logging.basicConfig(
+        format="{asctime} - {levelname} - {message}",
+        style="{",
+        datefmt="%Y-%m-%d %H:%M",
+        level=logging.INFO,
+    )
+
+
+def convert_from_unix_to_datetime(unix: str) -> datetime:
+    """Converts a unix timestamp to a regularly
+     formatted date"""
+
+    date = datetime.fromtimestamp(unix)
+
+    return date
+
+
+def convert_to_full_url(url: str) -> str:
+    """Checks if a url is valid and returns
+    the full url if it is not"""
+
+    if url.startswith("http"):
+        return url
+
+    return "https:" + url
+
+
+def get_sale_information(sales_dict: dict) -> list[dict]:
+    """Returns a list of dictionaries containing the
+    sales information"""
```

Comment:
This function handles both extracting genres and release dates. Could you split into smaller functions?

Reply:
I believe having the nested function calls of `get_release_date` and ` get_genres_from_url` are fine, as if I were to split them out as their own standalone functions, it may be harder to follow the flow of information.